galaxy-deps/values.yaml

```diff
@@ -2,6 +2,7 @@ postgresql:
   #- Whether to deploy the postgresl operator.
   #- In general, we recommend installing the operator globally in production.
   deploy: true
```

Comment:
Why have both?

Reply:
We had both in the Galaxy helm chart (https://github.com/galaxyproject/galaxy-helm/blob/00802d519485b78887d8a2ff9c16f9cf153dec2d/galaxy/values.yaml#L394) but maybe that doesn't make sense in this chart after all since it's just the operator, and no database. I'll update.